### PR TITLE
Feat/dynamic cli args

### PR DESCRIPTION
**Pull request by JulienChampagnol**

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -1,18 +1,7 @@
...
-    "nuxt-electron",
-    "vuetify-nuxt-module",
-    [
-      "@pinia/nuxt",
-      {
-        autoImports: ["storeToRefs", "defineStore"],
-      },
-    ],
-    "@nuxt/devtools",
-    "@vueuse/nuxt",
```
Comment (BotellaA) [inline nuxt.config.js:14]: Why removing it?

File: nuxt.config.js [hunk 2]
```diff
@@ -27,6 +16,10 @@ export default defineNuxtConfig({
         },
       },
     ],
+    disableDefaultOptions: true,
+  },
+  experimental: {
+    appManifest: false,
```
Comment (BotellaA) [inline nuxt.config.js:22]: Not sur this is useful

**Review by BotellaA — APPROVED** [review]

File: nuxt.config.js (hunk 1 above)
Comment (JulienChampagnol) [reply nuxt.config.js:14]: Already in ogw-f nuxt-config

**Pull request merged**